src/main/java/com/cricket/project/exception/PlayerException.java

```diff
@@ -0,0 +1,7 @@
+package com.cricket.project.exception;
+
+public class PlayerException extends Exception{
```

Comment:
The name can be more generic as it is being used everywhere in the code and not just the player related things